packages/wonder-blocks-tooltip/util/visibility-modifier.test.js

```diff
@@ -0,0 +1,185 @@
+// @flow
+import Popper from "popper.js";
+import * as Core from "@khanacademy/wonder-blocks-core";
+//import isObscured from "./is-obscured.js";
+import visibilityModifierDefaultConfig from "./visibility-modifier.js";
+import isObscured from "./is-obscured.js";
+
+jest.mock("@khanacademy/wonder-blocks-core");
+jest.mock("./is-obscured.js");
+
+describe("Visibility PopperJS Modifier", () => {
+    test("returns an enabled modifier configuration with expected order", () => {
+        // Arrange
+
+        // Act
+        const result = visibilityModifierDefaultConfig;
+
+        // Assert
+        expect(result).toMatchObject({
+            enabled: true,
+            fn: expect.any(Function),
+            order: Popper.Defaults.modifiers.hide.order + 1,
+        });
+    });
+
+    describe("#fn", () => {
+        // For all these tests, we don't need real elements, just things that
+        // look like real elements. We'll fake some known dimensions and verify
+        // that the results return what we want.
+        describe("Obscured by one or more scroll parents", () => {
+            test("Obscured horizontally only, is not visible", () => {
+                // Arrange
+                const psuedoAnchorElement = {
+                    getBoundingClientRect: () => ({
+                        top: 0,
+                        left: 0,
+                        right: 100,
+                        bottom: 100,
```

Comment:
The value of this is unimportant since we're mocking `Core.getElementIntersection` anyways, right?

Reply:
Yeah, I can add a comment to make that clear.

Reply:
In fact, they're a hangover from before I split `isObscured` into it's own method. Now that we're mocking that method out, we don't need this fake bounds rect call. :grin: